CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Unreleased
 
+## Templates
+- Triggers may now use templates which will be instantiated and added to the parent when active (like a node child).
+	<WhileActive>
+		<Circle ux:Generate="Template" Color="#AFA" Width="50" Height="50" Alignment="BottomRight"/>
+	</WhileActive>
+- Added templates to `NodeGroup`, which can now be used in `Each.TemplateSource` and `Instance.TemplateSource`
+- `Each`, using `TemplateSource`, will no longer respond to template changes after rooting. This was done to simplify the code, and to support alternate sources, and is a minor perf improvement. It's not likely to affect any code since it didn't work correctly, and there's no way in UX to modify templates after rooting.
```

Comment:
This refers to the list of templates itself, nothing ever responded to changes inside a template before. 

Unless preview had special support for `Each` I doubt this would make an impact. Template changes require a reify of the parent node at a minimum -- this would be throughout fuselibs, not just in `Each`.

The mechanism was also somewhat broken before, only working in some cases (not the most common ones either). Thus I felt it a safe change to make.